modules/osd-create-cluster-ccs.adoc

```diff
@@ -220,6 +220,11 @@ CIDR configurations cannot be changed later. Confirm your selections with your n
 You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
 ====
 +
+.. Provide administrator approval per your cluster update method:
+** Individual updates: If you select an update version that requires approval, provide an administrator’s acknowledgment and click *Approve and continue*.
```

Comment:
How do users know if the update version requires approval? I assume that the UI somehow indicates ?

Reply:
Yes, the UI indicates if the update version that is selected for the upgrade requires admin approval.